explorer-main

arsalanakhter opened the pull request.

joseph-cooke committed:
added export and source file directions

joseph-cooke committed:
updated cmake and included export

SerenaCaplins committed:
edits to the understanding the queue

SerenaCaplins committed:
edits to the understanding the queue

SerenaCaplins committed:
edits to the understanding the queue

SerenaCaplins committed:
moving files to keep to running jobs

SerenaCaplins committed:
moving files to keep to running jobs

SerenaCaplins committed:
remove AI page

SerenaCaplins committed:
added the link to slurm

SerenaCaplins committed:
change format of understanding the queue

SerenaCaplins committed:
Update understandingqueuing.md

SerenaCaplins committed:
Update homequota.md

SerenaCaplins committed:
Update homequota.md

arsalanakhter committed:
Fixed another link to storage options at the bottom of the page

arsalanakhter committed:
Merge branch 'explorer-main' into Rakshith2605-patch-2

arsalanakhter committed:
Merge pull request #349 from northeastern-rc/Rakshith2605-patch-2

page conda-rakshith

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
new user name

SerenaCaplins committed:
update parallel toolbox

SerenaCaplins committed:
change cluster name to explorer

arsalanakhter committed:
Update multigpu-partition-access.md for explorer

arsalanakhter committed:
Merge pull request #353 from northeastern-rc/globus-explorer

Update globus.md for Explorer

arsalanakhter committed:
Update transferringdata.md for Explorer

arsalanakhter committed:
Update mpi.md for explorer

This is an initial commit and more edits will be done

arsalanakhter committed:
Remove the module load command

arsalanakhter committed:
removed srun inside sbatch

arsalanakhter committed:
Merge pull request #356 from northeastern-rc/multigpu-explorer

Update multigpu-partition-access.md for explorer

SerenaCaplins committed:
Merge pull request #354 from northeastern-rc/reassure

Update transition.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Merge pull request #359 from northeastern-rc/courses_transition

add courses access to explorer transition.md

arsalanakhter committed:
Merge pull request #357 from northeastern-rc/data-transfer-for-explorer

Update transferringdata.md for Explorer

SerenaCaplins committed:
Update class_guide.md

SerenaCaplins committed:
Update class_guide.md

SerenaCaplins committed:
Update class_guide.md

SerenaCaplins committed:
Update index.md

arsalanakhter committed:
Fixed a line on available software and a conda link

arsalanakhter committed:
Fixed link on srun

arsalanakhter committed:
Fixed wording on OOD Under software applications to link it to RC website

arsalanakhter committed:
updated code-block for sbatch submission for better rendering

arsalanakhter committed:
fix the link to conda environment

SerenaCaplins committed:
removed OOD from software as it was mentioned in the new OOD section

SerenaCaplins committed:
changed gpu jobs to 1

SerenaCaplins committed:
Merge pull request #360 from northeastern-rc/fix_links

Update class_guide.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
change connectingtocluster

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
change header for containers index

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
courses

arsalanakhter committed:
Change "User Guides" To "Explorer User Guides"

SerenaCaplins committed:
Merge pull request #361 from northeastern-rc/explorifyit

changed titles for sections to highlight that this is explorer

SerenaCaplins committed:
Update class_guide.md

SerenaCaplins committed:
make bold

SerenaCaplins committed:
adding in course materials will be archived for one year

SerenaCaplins committed:
Update class_guide.md

SerenaCaplins committed:
Merge pull request #365 from northeastern-rc/idelbot

added a description of idle bot to the class guide

arsalanakhter committed:
Removed example scripts from the page

arsalanakhter committed:
Update homequota.md for use on explorer

arsalanakhter committed:
Update workquota.md for explorer

arsalanakhter committed:
Merge pull request #366 from northeastern-rc/arsalanakhter-patch-1

Removed example scripts from the page

arsalanakhter committed:
Merge pull request #368 from northeastern-rc/arsalanakhter-projects-explorer

Update workquota.md for explorer

arsalanakhter committed:
change singularity workflow to apptainer

arsalanakhter committed:
Merge branch 'explorer-main' into home-quota-on-explorer

arsalanakhter committed:
Merge branch 'explorer-main' into matlab_explorer

arsalanakhter committed:
changed matlab version in text and fixed link to OOD

arsalanakhter committed:
Merge pull request #367 from northeastern-rc/home-quota-on-explorer

Update homequota.md for use on explorer

arsalanakhter committed:
Update mpi.md for explorer, round 2

Akshit-H-Jain committed:
Update homequota.md

Updated the colons in the Note

Akshit-H-Jain committed:
Update homequota.md

Added more information on Home quota

Akshit-H-Jain committed:
Update scratchpurge.md

Changed /work to /projects.
Added a link to scratch policy page.

Akshit-H-Jain committed:
Update index.md

Changed work to projects in the card

SerenaCaplins committed:
Add files via upload

SerenaCaplins committed:
Update cheatsheet.md

SerenaCaplins committed:
Add files via upload

SerenaCaplins committed:
Merge pull request #355 from northeastern-rc/matlab_explorer

Matlab explorer

SerenaCaplins committed:
incorporate typos

SerenaCaplins committed:
Merge branch 'explorer-main' into fix_cheatsheet

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Merge pull request #373 from northeastern-rc/fix_cheatsheet

fix error in xfer node command

SerenaCaplins committed:
there's an e

SerenaCaplins committed:
Merge branch 'explorer-main' into transferringfiles

SerenaCaplins committed:
Merge pull request #374 from northeastern-rc/transferringfiles

Adding a note about the hostname for xfer

SerenaCaplins committed:
Update homequota.md

SerenaCaplins committed:
Merge pull request #376 from northeastern-rc/SerenaCaplins-patch-1

Update homequota.md

arsalanakhter committed:
Merge pull request #372 from northeastern-rc/bestpractices_indexpage_explorer-1

Update index.md

arsalanakhter committed:
Merge pull request #371 from northeastern-rc/ScratchDirectoryPurge_explorer

Update scratchpurge.md

arsalanakhter committed:
Merge pull request #370 from northeastern-rc/akshit-patch1-1

Update homequota.md

SufurElite committed:
added seff and historical seff information

Gayathri1331 committed:
Updated conda.md: Modified conda create command and added reference link for export

Updated the conda create command without Anaconda. 

Provided an optional version with Anaconda for users needing pre-installed packages.

Added a reference link for exporting Conda environments.

arsalanakhter committed:
Mentioned that building whole anaconda can take more time

SerenaCaplins committed:
Update faqs-new.md

SerenaCaplins committed:
Update class_guide.md

SerenaCaplins committed:
Update mpi.md

SerenaCaplins committed:
Create module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update faqs-new.md

SufurElite committed:
an empty commit

SufurElite committed:
an empty commit

SufurElite committed:
an empty commit

SufurElite committed:
empty commit

SufurElite committed:
empty commit

SufurElite committed:
empty commit

Gayathri1331 committed:
Update slurmarray - issue #263.md

Addressing GitHub issue #263.
Added note that #SBATCH resource directives apply to each individual task in a Slurm job array, not to the array as a whole.

Gayathri1331 committed:
Update slurmarray - issue #263.md

Addressing GitHub issue #263.
Added note that #SBATCH resource directives apply to each individual task in a Slurm job array, not to the array as a whole.

Gayathri1331 committed:
Update slurmarray - issue #263.md

Addressing GitHub issue #263.
Added note that #SBATCH resource directives apply to each individual task in a Slurm job array, not to the array as a whole.

Akshit-H-Jain committed:
Updated MultiGPU document with latest process

SerenaCaplins committed:
Update faqs-new.md

SerenaCaplins committed:
Update faqs-new.md

SerenaCaplins committed:
Merge pull request #382 from northeastern-rc/faqs_andcleanup

Update faqs-new.md

joseph-cooke committed:
h200 quickstart guide

arsalanakhter committed:
Add a card in index.md to link to H200 guide

arsalanakhter committed:
Modify the link to accessing GPUS

arsalanakhter committed:
Modify the card mentioning H200s

arsalanakhter committed:
Minor language modification around the link to GPU access

tketchem14 committed:
Update quickstart-h200.md

Made a few minor changes.

joseph-cooke committed:
Update quickstart-h200.md

sbatch callout

joseph-cooke committed:
Merge pull request #390 from northeastern-rc/h200-quickstart

h200 quickstart guide

arsalanakhter committed:
Add MAPO Announcement bar 2025

arsalanakhter committed:
Add MAPO Announcement bar 2025

arsalanakhter committed:
Add MAPO Announcement bar 2025

arsalanakhter committed:
MAPO Announcement 2025

arsalanakhter committed:
Merge pull request #392 from northeastern-rc/mapo-2025-1

MAPO Announcement 2025

SerenaCaplins committed:
Merge branch 'explorer-main' into module_tutorials

SerenaCaplins committed:
Update modules.md

SerenaCaplins committed:
Update modules.md

SerenaCaplins committed:
Update module_list.md

arsalanakhter committed:
Remove MAPO Announcement banner

arsalanakhter committed:
Remove MAPO Announcement banner

arsalanakhter committed:
Remove MAPO Announcement banner

arsalanakhter committed:
Remove MAPO for Explorer

SufurElite committed:
removed broken and redundant discovery storage link

SufurElite committed:
removed random non-existent getting started link

SufurElite committed:
removed nonexistent welcome index reference

SufurElite committed:
removed nonexistent hardware link

SufurElite committed:
removed link to nonexist tutorialsandtraining/index

SufurElite committed:
removed nonexistent misc annual-shutdown link

SufurElite committed:
removed link to jobscheduling which was removed on Nov 6,2024 for being AI

SufurElite committed:
ignore the non-consecutive header problem and update the location of cps ood

SufurElite committed:
removed multiple faqs with broken linkls

SufurElite committed:
the welcome services were moved to the website

SufurElite committed:
reference our website since 'data-storage' no longer exists on the RTD

SufurElite committed:
moved the link again and then also removed the non-existent apptainer term

SufurElite committed:
changed the top of the file from the name of the file so that myst has no ambiguity

SufurElite committed:
updated the way we link to a different page so it's correct

SufurElite committed:
update the class_guide sbatch link so it actually works

SufurElite committed:
data storage page no longer exists and we now point to our website for info

SufurElite committed:
Removed broken link to file-explorer, a file deleted ages ago that was part of the using-ood page

SufurElite committed:
update so the name of the section inside the doc to remove ambiguity for myst

SufurElite committed:
removed a reference to the command line page (usingbash.md), which was removed

SufurElite committed:
Removed broken open ondemand interactive link and referenced our website instead

SufurElite committed:
we don't have a using-ood on the rtd anymore, this should point to the website

SufurElite committed:
added a doc reference to the index.md of connecting to cluster so that systemwide modules points to somewhere

SufurElite committed:
removed the creating module files subsection

SufurElite committed:
Updated OOD reference point for r module page

SufurElite committed:
file-removed not immediately supported by sphinx

SufurElite committed:
removed the premature ::: which closed the grid

SufurElite committed:
update range for gpujobsubmission

arsalanakhter committed:
Merge pull request #395 from northeastern-rc/remove-MAPO-explorer

Remove MAPO for Explorer

SufurElite committed:
Merge branch 'explorer-main' into tidy-explorer

SufurElite committed:
Remove YAML approach for MyST directive parsing

SufurElite committed:
Merge pull request #396 from northeastern-rc/tidy-explorer

Tidying Up Explorer (fixing the errors in explorer-main)

SufurElite committed:
Fixed spelling error

SufurElite committed:
project management-init

SufurElite committed:
updated some wording

SufurElite committed:
updated number

SufurElite committed:
added note providing clarification on input

arsalanakhter committed:
More edits

arsalanakhter committed:
More edits

arsalanakhter committed:
adding gpu-interactive on explorer-main

arsalanakhter committed:
Additional edits

SerenaCaplins committed:
Update workquota.md

SerenaCaplins committed:
Merge pull request #402 from northeastern-rc/SerenaCaplins-patch-1

Update workquota.md

SerenaCaplins committed:
Merge branch 'explorer-main' into module_tutorials

arsalanakhter committed:
Merge pull request #401 from northeastern-rc/gpu-interactive-explorer-main

adding gpu-interactive on explorer-main

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
new table using script to update

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Merge branch 'explorer-main' into module_tutorials

SerenaCaplins committed:
add versions

SerenaCaplins committed:
adding in more versions

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
adding in modules that dont have a repo

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
removing some non-human generated text from modules.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
make containers a ref link

SerenaCaplins committed:
Create containers_list.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update index.md

SerenaCaplins committed:
Update containers_list.md

SerenaCaplins committed:
Update containers_list.md

SerenaCaplins committed:
Update containers_list.md

SerenaCaplins committed:
Update r.md

SerenaCaplins committed:
Update containers_list.md

SufurElite committed:
fix whitespace

SufurElite committed:
Merge branch 'explorer-main' into user-management

SufurElite committed:
one more spot where it was project-management that needed to be updated

arsalanakhter committed:
Update index.md to modify the title

arsalanakhter committed:
Merge pull request #397 from northeastern-rc/user-management

Project Management

Akshit-H-Jain committed:
Updated terminology.md for spellings

SerenaCaplins committed:
Merge pull request #403 from northeastern-rc/manageprojects_updates

Updated terminology.md for spellings

SerenaCaplins committed:
Update containers_list.md

SerenaCaplins committed:
Update module_list.md

SerenaCaplins committed:
Merge branch 'explorer-main' into module_tutorials

SerenaCaplins committed:
Merge pull request #383 from northeastern-rc/module_tutorials

Create module_list.md

SerenaCaplins committed:
Update commands.md

adding commands "groups" and "getfacl"

SerenaCaplins committed:
Merge pull request #404 from northeastern-rc/adding_commands_project

Update commands.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Update transition.md

SerenaCaplins committed:
Update transition.md

arsalanakhter committed:
A code block wasn't working, plus, Fix formatting and punctuation in transition.md 

Corrected capitalization, punctuation, and formatting issues throughout the document for clarity and consistency.

arsalanakhter committed:
Merge pull request #405 from northeastern-rc/updating_quickstart

Update transition.md

SufurElite committed:
added an idlebot page and info on historical-seff + gpu-logs

arsalanakhter committed:
reset clusterusage.md

arsalanakhter committed:
Merge pull request #406 from northeastern-rc/explorer-main-rebased

Explorer main rebased

SufurElite committed:
Merge pull request #407 from northeastern-rc/historical_seff

added an idlebot page and info on historical-seff + gpu-logs

arsalanakhter committed:
Merge branch 'master' into explorer-main

arsalanakhter committed:
remove referrence to non-existent containers file

arsalanakhter committed:
remove main heading from module list

arsalanakhter committed:
Add faq around sponsored accounts

arsalanakhter committed:
Reformat FAQ around using IB

joseph-cooke commented on docs/source/best-practices/scratchpurge.md:
```diff
@@ -33,23 +33,23 @@ You will need to copy the script below to a text file (e.g., filemover.sh) and s
 :::
 
-2. On the day of the /scratch purge you will not be able to write job outputs to /scratch. Please edit your sbatch scripts to write outputs to /work or to /home.
+2. On the day of the /scratch purge you will not be able to write job outputs to /scratch. Please edit your sbatch scripts to write outputs to /projects or to /home.
```
You will be able to write to your /scratch directory during the purges.

arsalanakhter replied:
Removed this sentence

joseph-cooke commented on docs/source/best-practices/scratchpurge.md:
```diff
@@ -64,7 +64,7 @@ Files and directories are removed during a /scratch purge. This saves space for
 ## How do I know /scratch is usable again following the /scratch purge?
 
-We will message via email when the scratch purge is finished. You can also check for updates on the Research Computing [website](https://rc.northeastern.edu/status-updates/). Additionaly, you can also check if the below commands work without error:
+We will message via email when the scratch purge is finished. You can also check for updates on the Research Computing [website](https://rc.northeastern.edu/status-updates/). Additionaly, you can also check if the below commands executes without error:
```
This needs to be removed because we do not email everyone after the purge is complete.

arsalanakhter replied:
removed

joseph-cooke commented on docs/source/classroom/class_guide.md:
```diff
@@ -62,54 +64,50 @@ The resources available in the courses/courses-gpu partitions can be queried wit
 :::
 
+:::{important}
+The compute resources for courses are shared across all courses each term. [We monitor their usage daily](../best-practices/clusterusage.md). **We send out email notifications to users who are idle on courses-gpu for one hour.** We highly recommend ending jobs when your work has finished as this frees up the resource for other students. When all students do this, it increases the availability of  ...
```
Do we end the jobs in these cases? If so, we need to change the wording in this to indicate that after an hour, the job will be terminated.

arsalanakhter replied:
added that idle jobs on courses-gpu are terminated after an hour

joseph-cooke commented on docs/source/classroom/cps_ood.md:
```diff
@@ -15,19 +15,19 @@ Due to problems with launching OOD on **Safari**, we recommend using **Google Ch
+1. In a web browser, go to <http://ood.explorer.northeastern.edu>. Login with your NU credentials.
 
-1. Under the **Courses** menu, select your Class Name (For example: **ALY6080 JupyterLab**).
+2. Under the **Courses** menu, select your Class Name (For example: **ALY6080 JupyterLab**).
```
We may want to suggest that they can use the general unless all CPS courses request custom apps.

arsalanakhter replied:
Interestingly, the file `docs/source/classroom/cps_ood.md` is not part of the toctree or index, so it cannot be reached from within the RTD. (Google might still index it). Maybe we should delete this file, but that might cause a merge conflict in this branch. Hopefully, we can address it in a subsequent cleanup effort.

SerenaCaplins reviewed (approved):
I reviewed all pages (though review was brief).

It looks really good and thank you for putting together!

joseph-cooke reviewed (approved):
Thank you for getting this together. Just a couple of comments. Thank you!!

arsalanakhter committed:
Revise scratch purge guidelines

Updated instructions regarding scratch purge processes, including checkpointing and directory retention.

arsalanakhter committed:
Update idle job notification policy in class guide

added that jobs are terminated
